### ISSUE TITLE
Request to change some colors of the States + change the Classic to be the default theme

### ISSUE THREAD

**Issue by anna-geller** [kestra-io/kestra#5073]
### Feature description

Sorry for quite a bit back and forth on this @MilosPaunovic - we discussed with Ludo and Nico to make sure the states that require some action or show an anomaly (like Paused and Created) are displayed in a more prominent way 👍 

You can keep the `Classic` red-green as the default color palette as @loicmathieu and @tchiotludo requested because indeed some people will prefer that one for production and we currently have no way for admins to set that default. 

We can name the "Purple" theme as "Colorblind-friendly" theme

So in summary, remaining TODOs here for 0.19.0:
- [x] change the color palette once Nico adjusts it tomorrow morning https://www.figma.com/design/ew0uXk0NRXJ2NBBJTNe2n1/01_UI?node-id=3041-29504&t=TjUzqcFIHV3Wqehz-4 
- [x] Change the default theme to be the `Classic` red-green one 
- [x] Change the name of the second theme in Settings to be the `Colorblind-friendly` (no longer the default theme and no longer named `Default` theme)

**Comment by MilosPaunovic** [comment]
@Nico-Kestra Do you mind editing this format and passing it back to me once you adjust it?

```
{
    default: {
        light: {
            executions: {
                CANCELLED: "#fd9297",
                CREATED: "#3991ff",
                FAILED: "#fd7278",
                KILLED: "#bda85d",
                KILLING: "#fde89d",
                PAUSED: "#a6a4ca",
                QUEUED: "#7e719f",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#21ce9c",
                WARNING: "#eeae7e",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#fd7278",
                INFO: "#21ce9c",
                TRACE: "#9a8eb4",
                WARN: "#f3c4a1",
            },
        },
        dark: {
            executions: {
                CANCELLED: "#fd9297",
                CREATED: "#3991ff",
                FAILED: "#fd7278",
                KILLED: "#bda85d",
                KILLING: "#fde89d",
                PAUSED: "#a6a4ca",
                QUEUED: "#7e719f",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#21ce9c",
                WARNING: "#eeae7e",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#fd7278",
                INFO: "#21ce9c",
                TRACE: "#9a8eb4",
                WARN: "#f3c4a1",
            },
        },
    },
    purple: {
        light: {
            executions: {
                CANCELLED: "#fec9cb",
                CREATED: "#3991ff",
                FAILED: "#ff4bbd",
                KILLED: "#bda85d",
                KILLING: "#fde89d",
                PAUSED: "#a6a4ca",
                QUEUED: "#9a8eb4",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#8c4bff",
                WARNING: "#cf5df9",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#ff4bbd",
                INFO: "#8c4bff",
                TRACE: "#a6a4ca",
                WARN: "#cf5df9",
            },
        },
        dark: {
            executions: {
                CANCELLED: "#fec9f5",
                CREATED: "#3991ff",
                FAILED: "#ff41b9",
                KILLED: "#fdedb3",
                KILLING: "#fce07c",
                PAUSED: "#d1cfe9",
                QUEUED: "#e5e4f7",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#9470ff",
                WARNING: "#cc4af8",
            },
            logs: {
                DEBUG: "#7fbbff",
                ERROR: "#ff41b9",
                INFO: "#9470ff",
                TRACE: "#d1cfe9",
                WARN: "#cc4af8",
            },
        },
    },
}
```

`default` is the red-green one, `purple` is the purple one, and key for `light` means product is in light mode, and `dark` means the product is in dark mode.

@anna-geller Would you be okay with this naming scheme:
![image](https://github.com/user-attachments/assets/b48b87d8-9897-4485-9e96-efa5d79bed4a)

And hovering on `i` icon next to heading would show this:
![image](https://github.com/user-attachments/assets/35814b68-7b0e-4de7-b168-6be8da155046)

That way we have options to add as much variations of regular and color blind safe themes as we want.

**Comment by anna-geller** [comment]
Milos, let's not invent any acronym like CBS 😄  either keep "Colorblind", "Colorblind-friendly" or just "Kestra" or "Purple"

**Comment by MilosPaunovic** [comment]
This is all now properly sorted and pushed to develop, only thing left is for @Nico-Kestra to change and get back this object to me so I can get it up and we have improved coloring scheme.

```
{
    default: {
        light: {
            executions: {
                CANCELLED: "#fd9297",
                CREATED: "#3991ff",
                FAILED: "#fd7278",
                KILLED: "#bda85d",
                KILLING: "#fde89d",
                PAUSED: "#a6a4ca",
                QUEUED: "#7e719f",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#21ce9c",
                WARNING: "#eeae7e",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#fd7278",
                INFO: "#21ce9c",
                TRACE: "#9a8eb4",
                WARN: "#f3c4a1",
            },
        },
        dark: {
            executions: {
                CANCELLED: "#fd9297",
                CREATED: "#3991ff",
                FAILED: "#fd7278",
                KILLED: "#bda85d",
                KILLING: "#fde89d",
                PAUSED: "#a6a4ca",
                QUEUED: "#7e719f",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#21ce9c",
                WARNING: "#eeae7e",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#fd7278",
                INFO: "#21ce9c",
                TRACE: "#9a8eb4",
                WARN: "#f3c4a1",
            },
        },
    },
    purple: {
        light: {
            executions: {
                CANCELLED: "#fec9cb",
                CREATED: "#3991ff",
                FAILED: "#ff4bbd",
                KILLED: "#bda85d",
                KILLING: "#fde89d",
                PAUSED: "#a6a4ca",
                QUEUED: "#9a8eb4",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#8c4bff",
                WARNING: "#cf5df9",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#ff4bbd",
                INFO: "#8c4bff",
                TRACE: "#a6a4ca",
                WARN: "#cf5df9",
            },
        },
        dark: {
            executions: {
                CANCELLED: "#fec9f5",
                CREATED: "#3991ff",
                FAILED: "#ff41b9",
                KILLED: "#fdedb3",
                KILLING: "#fce07c",
                PAUSED: "#d1cfe9",
                QUEUED: "#e5e4f7",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#9470ff",
                WARNING: "#cc4af8",
            },
            logs: {
                DEBUG: "#7fbbff",
                ERROR: "#ff41b9",
                INFO: "#9470ff",
                TRACE: "#d1cfe9",
                WARN: "#cc4af8",
            },
        },
    },
}
```

`default` is the red-green one, `purple` is the purple one, and key for `light` means product is in light mode, and `dark` means the product is in dark mode.

**Comment by Nico-Kestra** [comment]
For color mode name i vote for "Kestra"

it's the last version:
<img width="1947" alt="Capture d’écran 2024-09-24 à 22 37 06" src="https://github.com/user-attachments/assets/ce6438f5-1d7e-40ab-bb3e-eedd9306208b">
```
`
{
    default: {
        light: {
            executions: {
                CANCELLED: "#d1cfe9",
                CREATED: "#fec9cb",
                FAILED: "#fd7278",
                KILLED: "#e5e4f7",
                KILLING: "#d1cfe9",
                PAUSED: "#fce07c",
                QUEUED: "#fdedb3",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#02be8a",
                WARNING: "#e9985b",
            },
            logs: {
                DEBUG: "#7fbbff",
                ERROR: "#fd7278",
                INFO: "#21ce9c",
                TRACE: "#d1cfe9",
                WARN: "#eeae7e",
            },
        },
        dark: {
            executions: {
                CANCELLED: "#9a8eb4",
                CREATED: "#fd9297",
                FAILED: "#fd7278",
                KILLED: "#7e719f",
                KILLING: "#a6a4ca",
                PAUSED: "#fde89d",
                QUEUED: "#bda85d",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#21ce9c",
                WARNING: "#eeae7e",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#fd7278",
                INFO: "#21ce9c",
                TRACE: "#9a8eb4",
                WARN: "#f3c4a1",
            },
        },
    },
    purple: {
        light: {
            executions: {
                CANCELLED: "#d1cfe9",
                CREATED: "#fec9f5",
                FAILED: "#ff41b9",
                KILLED: "#e5e4f7",
                KILLING: "#d1cfe9",
                PAUSED: "#fce07c",
                QUEUED: "#fdedb3",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#9470ff",
                WARNING: "#e9985b",
            },
            logs: {
                DEBUG: "#7fbbff",
                ERROR: "#ff41b9",
                INFO: "#9470ff",
                TRACE: "#d1cfe9",
                WARN: "#eeae7e",
            },
        },
        dark: {
              CANCELLED: "#a6a4ca",
                CREATED: "#fec9cb",
                FAILED: "#ff4bbd",
                KILLED: "#9a8eb4",
                KILLING: "#a6a4ca",
                PAUSED: "#fde89d",
                QUEUED: "#bda85d",
                RESTARTED: "#c7f0ff",
                RETRIED: "#a2cdff",
                RETRYING: "#7fbbff",
                RUNNING: "#5bb8ff",
                SUCCESS: "#8c4bff",
                WARNING: "#eeae7e",
            },
            logs: {
                DEBUG: "#3991ff",
                ERROR: "#ff4bbd",
                INFO: "#8c4bff",
                TRACE: "#a6a4ca",
                WARN: "#f3c4a1",
            },
        },
    },
}
```

**Comment by Nico-Kestra** [comment]
I would place this form here, WDYT?

<img width="1296" alt="Capture d’écran 2024-09-24 à 22 57 07" src="https://github.com/user-attachments/assets/b78eee93-4fa7-48df-a00f-10946e9378b2">

**Comment by Nico-Kestra** [comment]
And a quick question: you asked me about the colors for the charts, but how are you managing the tags on the other pages to ensure visual consistency?
example:
<img width="262" alt="Capture d’écran 2024-09-24 à 23 04 26" src="https://github.com/user-attachments/assets/d87961f0-efad-4302-b236-f36ba9e14610">

**Comment by MilosPaunovic** [comment]
Amended the coloring scheme and moved the select box in setting to new location, this now is ready for another review @anna-geller.

About the last question @Nico-Kestra, I have not touched anything color-wise apart from thing on the dashboard in this PR. If we need to change it somewhere else, we could open separate issue for it.
